### PR TITLE
Simd blog

### PR DESCRIPTION
**Pull request by pnowojski**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,37 +1,3 @@
-## Welcome to GitHub Pages
+## Welcome to Teradata Presto GitHub Pages
```
Comment (ilfrin) [inline index.md]: change to "## Teradata Presto Technical Blogs" or something like that

File: index.md [hunk 2]
```diff
@@ -1,37 +1,3 @@
...
-For more details see [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/).
-
-### Jekyll Themes
-
-Your Pages site will use the layout and styles from the Jekyll theme you have selected in your [repository settings](https://github.com/Teradata/presto-tech/settings). The name of this theme is saved in the Jekyll `_config.yml` configuration file.
-
-### Support or Contact
-
-Having trouble with Pages? Check out our [documentation](https://help.github.com/categories/github-pages-basics/) or [contact support](https://github.com/contact) and we’ll help you sort it out.
+[JAVA and SIMD Blog post](blogs/simd.md)
```
Comment (ilfrin) [inline index.md]: how will we add more? is there a mechanism for it or we just add lines here? what about generating a menu etc? does that work somehow?

**Review by ilfrin — CHANGES_REQUESTED** [review]
I assume the formatting it good, haven't checked it, and that the general layout and look&feel is good

also ask Grzegorz to add his content here as well

File: index.md (hunk 1 above)
Comment (ilfrin) [reply index.md]: maybe we should add a subtitle or a paragraph like:
"This is a blog space for the Teradata Presto engineering team to share our thoughts and findings on the various software engineering topics we come across during the development of Presto - the open source distributed analytical SQL engine (https://github.com/prestodb/presto). Topics areas include, but are not limited to: Java performance tuning, database internals, SQL optimizers, etc"

Comment (ilfrin) [reply index.md]: or something like that, so it looks good and describes our context and intentions

Comment (pnowojski) [reply index.md]: done

File: index.md (hunk 2 above)
Comment (pnowojski) [reply index.md]: I couldn't find a way to generate a menu, maybe Grzesiek will improve it. For now, we can add it manually.

**Pull request merged**